book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] Great job working on the functionality for this method but I think the rental is related with both the person and the book so I can see it would be better if you implement it accordingly so you can add something like this since the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. ✅🚀💯

```ruby
 def add_rental(person, date)
    Rental.new(date, self, person)
  end
```